rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -2522,14 +2528,34 @@ private int sourceLengthOfString(ConstantExpression expr) {
         return lengthAccordingToAst;
     }
 
-    private static @Nullable Integer getInsideParenthesesLevel(ASTNode node) {
+    private @Nullable Integer getInsideParenthesesLevel(ASTNode node) {
         Object rawIpl = node.getNodeMetaData("_INSIDE_PARENTHESES_LEVEL");
         if (rawIpl instanceof AtomicInteger) {
             // On Java 11 and newer _INSIDE_PARENTHESES_LEVEL is an AtomicInteger
             return ((AtomicInteger) rawIpl).get();
-        } else {
+        } else if (rawIpl instanceof Integer) {
             // On Java 8 _INSIDE_PARENTHESES_LEVEL is a regular Integer
             return (Integer) rawIpl;
+        } else if (node instanceof MethodCallExpression) {
+            MethodCallExpression expr = (MethodCallExpression) node;
+            int childBegin = cursor;
+            if (expr.getObjectExpression().getLineNumber() != expr.getLineNumber()) {
```

Comment:
```suggestion
            if (expr.getObjectExpression().getLineNumber() >= expr.getLineNumber()) {
```
Just to catch some really weird/unexpected line numbering.